tests/foreman/cli/test_activationkey.py

```diff
@@ -1742,8 +1742,7 @@ def test_syspurpose_end_to_end(
     assert host['subscription-information']['system-purpose']['purpose-usage'] == "test-usage2"
     assert host['subscription-information']['system-purpose']['service-level'] == "Self-Support2"
 
-    # Unregister host
-    target_sat.cli.Host.subscription_unregister({'host': rhel_contenthost.hostname})
+    rhel_contenthost.unregister()
```

Comment:
We can keep comment there as it is.
```suggestion
    # Unregister host
    rhel_contenthost.unregister()
```

Reply:
unnecesarry comment, you can clearly see what is beign done in the code